Sources/CardPayments/Models/CardResult.swift

```diff
@@ -9,10 +9,9 @@ public struct CardResult {
     /// The order ID associated with the transaction
     public let orderID: String
 
-    /// This is the deep link url returned from 3DS authentication
-    @_documentation(visibility: private)
-    public let deepLinkURL: URL?
-    
+    /// status of the order
+    public let status: String?
+
     /// Liability shift value returned from 3DS verification
-    public let liabilityShift: String?
+    public let threeDSecureAttempted: Bool
```

Comment:
```suggestion
    public let didAttemptThreeDSecureAuthentication: Bool
```
super nit: An `is` prefix may be more strictly in line with naming conventions for `Bool` properties, but "did" here reads well IMO, even though that naming convention is more common with delegate methods.

Reply:
I like it! I'd like to hear what others say about it. We also have the same property for CardVaultResult in vault without purchase PR: https://github.com/paypal/paypal-ios/blob/3e0d2636c43b4b7f8d7d55421d59bf4c550f9d51/Sources/CardPayments/Models/CardVaultResult.swift#L16

Reply:
I like it, agree with `didAttemptThreeDSecureAuthentication`